reference/checkouts.v3.yml

```diff
@@ -987,11 +961,9 @@ definitions:
                 id:
```

Comment:
{
          "discounted_amount": 0.24,
          "id": "977dc259-17df-4811-b300-2a124b42cf83"
        }
Based on the payload, `discounted_amount` supports format:double and `id` is a string.